### PR TITLE
cluster: check permission for deploy/data dir for exist clusters

### LINKED ISSUE #1062: before upgrade check owner and Authority of deploy and data directory
## Feature Request
before upgrade check owner and Authority deploy and data directory
**Is your feature request related to a problem? Please describe:**

The permissions of the actual user environment may be changed by mistake after running for a period of time, so it is best to check the users and permissions before upgrading.
**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by AstroProfundis**
### What problem does this PR solve? 
Close #1062 

### What is changed and how it works?
Add a test to `touch` an empty file in `data_dir` and `deploy_dir` if checking an exist cluster

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
图片

Release notes:

```release-note
cluster: check write permission for deploy_dir and data_dir for exist clusters
```

### CODE REVIEW

File: components/cluster/command/check.go [hunk 1]
```diff
@@ -174,137 +175,161 @@ func checkSystemInfo(s *cliutil.SSHConnectionProps, topo *spec.Specification, gO
 				downloadTasks = append(downloadTasks, t0)
 			}
 
-			if _, found := uniqueHosts[inst.GetHost()]; found {
+			if _, found := uniqueInsts[inst.ID()]; found {
```
Comment (lucklove) [inline components/cluster/command/check.go:178]: I found that this is redundant because `instt.ID()` will never present twice.
